src/librustc_typeck/diagnostics.rs

```diff
@@ -1640,7 +1640,8 @@ register_diagnostics! {
     E0323, // implemented an associated const when another trait item expected
     E0324, // implemented a method when another trait item expected
     E0325, // implemented an associated type when another trait item expected
-    E0327, // referred to method instead of constant in match pattern
+    //E0327, // referred to method instead of constant in match pattern
```

Comment:
I believe the codes themselves should never be removed (or commented out, because then error uniqueness check won’t fire), but I’m fine with adding a comment about this code being for an unreachable error. At least I couldn’t find a way to make this error code to be fired.